README.md

```diff
@@ -1,81 +1,69 @@
-
-## Requires Nvim 0.5 HEAD
-
-While Nvim LSP undergoes development, the configs in this repo assume that you
-are using the latest [Nvim HEAD/nightly build](https://github.com/neovim/neovim/releases/tag/nightly).
+This repo is :
 
-Update Nvim and nvim-lspconfig before reporting an issue.
+* a place for Nvim LSP users to collaboratively provide starting 
+  examples for the many LSP backends out there
+* a reference for the current best practice (or most popular) regarding 
+  choice of server, setup, etc.
 
-## Contributions
-
-It's up to you to send improvements so that these configs align with current
-best practices for a given language.
-
-1. Read [CONTRIBUTING.md](CONTRIBUTING.md).
-   Ask questions in [Neovim Gitter](https://gitter.im/neovim/neovim).
-2. Choose a language from [the coc.nvim wiki](https://github.com/neoclide/coc.nvim/wiki/Language-servers) or
-  [emacs-lsp](https://github.com/emacs-lsp/lsp-mode#supported-languages).
-3. Create a new file at `lua/lspconfig/SERVER_NAME.lua`.
-   - Copy an [existing config](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/)
-     to get started. Most configs are simple. For an extensive example see
-     [texlab.lua](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/texlab.lua).
+You do not need nvim-lspconfig to use neovim's built-in language server client, this repo
+just handles the configuration for you to make it easy
 
 ## Install
 
-- Requires [Nvim HEAD/nightly](https://github.com/neovim/neovim/releases/tag/nightly) (v0.5 prerelease).
-- nvim-lspconfig is just a plugin. Install it like any other Vim plugin, e.g. with [vim-plug](https://github.com/junegunn/vim-plug):
+* Requires [Nvim HEAD/nightly](https://github.com/neovim/neovim/releases/tag/nightly) (v0.5 prerelease). The configs in this repo
+  assume that you are using the latest [Nvim HEAD/nightly build](https://github.com/neovim/neovim/releases/tag/nightly). 
+  Update Nvim and nvim-lspconfig before reporting an issue.
+
+* nvim-lspconfig is just a plugin. Install it like any other Vim plugin, e.g. with [vim-plug](https://github.com/junegunn/vim-plug):
   ```
   :Plug 'neovim/nvim-lspconfig'
   ```
-- Call `:packadd nvim-lspconfig` in your vimrc if you installed nvim-lspconfig to `'packpath'` or if you use a package manager such as minpac.
-
-## Usage
-
-Each config provides a `setup()` function, to initialize the server with
-reasonable defaults and some server-specific things like commands or different
-diagnostics.
-
-```lua
-vim.cmd('packadd nvim-lspconfig')  -- If installed as a Vim "package".
-require'lspconfig'.<config>.setup{name=…, settings = {…}, …}
-```
 
-If you want to add this to your vimrc, you will need to enclose it in a `lua` block.
+Note: if you installed nvim-lspconfig to `'packpath'` you need to call 
+`:packadd nvim-lspconfig` in your vimrc.  This is the case for only 
+some package managers, such as minpac.  You do not need to do this 
+step with vim-plug or packer.
```

Comment:
You also need to do `packadd nvim-lspconfig` if you specify `opt = true` for packer.nvim.

Reply:
That's a really good point, will fix